docs/index.rst

```diff
@@ -10,6 +10,17 @@ parallelize, Just-In-Time compile to GPU/TPU, and more.
    Please see `Parallelism with JAX <https://jax.readthedocs.io/en/latest/notebooks/Parallelism_with_JAX.html>`_ tutorial and the :ref:`jax-array-migration`
    guide for more information.
 
+.. .. toctree::
+..    :maxdepth: 1
+..    :caption: User Specific Guides
+
+User Specific Guides
+********************
+
```

Comment:
I think here is where you could use a hidden toctree to correctly index these guides